llama_index/llms/ollama.py

```diff
@@ -111,6 +111,7 @@ def stream_complete(self, prompt: str, **kwargs: Any) -> CompletionResponseGen:
                 "Please install requests with `pip install requests`"
             )
         all_kwargs = self._get_all_kwargs(**kwargs)
+        del all_kwargs["formatted"]  # ollama throws 400 if it receives this option
```

Comment:
This is accessed just two lines later, and will fail.

Let's change below this to pop instead of get?